src/woo_publications/api/models.py

```diff
@@ -0,0 +1,69 @@
+import binascii
+import os
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from django_jsonform.models.fields import JSONField
+from phonenumber_field.modelfields import PhoneNumberField
+
+from .constants import PermissionOptions
+
+PERMISSION_SCHEMA = {
+    "type": "list",
+    "items": {
+        "type": "string",
+        "choices": [
+            {"title": label, "value": value}
+            for value, label in PermissionOptions.choices
+        ],
+        "widget": "multiselect",
+    },
+}
+
+
+class TokenAuth(models.Model):
+    token = models.CharField(_("token"), max_length=40, primary_key=True)
```

Comment:
You cannot use this as a primary key because then it shows up in the URLs in the admin interface, and thus in the server access logs.